### PR TITLE
fix(CommunitySettings): Process statuses for members model

### PR DESCRIPTION
**Pull request by borismelnik**
Closes: #6132

### What does the PR do

Process statuses for members model

### Affected areas

Community overview

### Screenshot of functionality
Снимок экрана 2022-07-04 в 16 55 15

### CODE REVIEW

File: ui/app/AppLayouts/Chat/panels/communities/CommunityMembersSettingsPanel.qml [hunk 1]
```diff
@@ -95,7 +95,8 @@ SettingsPageLayout {
                 id: memberItem
 
                 readonly property bool itsMe: model.pubKey.toLowerCase() === userProfile.pubKey.toLowerCase()
-                readonly property bool isOnline: model.onlineStatus === Constants.userStatus.online
+                readonly property bool isOnline: memberItem.itsMe ? true
```
Comment (osmaczko) [inline ui/app/AppLayouts/Chat/panels/communities/CommunityMembersSettingsPanel.qml:98]: This condition seems to be invalid.  Assumption that "me" is always online is probably wrong :thinking:

Comment (borismelnik) [reply ui/app/AppLayouts/Chat/panels/communities/CommunityMembersSettingsPanel.qml:98]: @osmaczko actually I don't know how it should be here. I can process every each status or process `online/offline`. There was be used just `online`. What do you think?

Comment (osmaczko) [reply ui/app/AppLayouts/Chat/panels/communities/CommunityMembersSettingsPanel.qml:98]: Not sure what you mean.. `membersModel` should be already populated with proper status, no extra logic is needed here.

**Review by osmaczko — APPROVED** [review]

**Review by jrainville — COMMENTED** [review]
I feel like some of the changes here are the same as @endulab 's PR https://github.com/status-im/status-desktop/pull/6058

**Comment by endulab** [comment]
> I feel like some of the changes here are the same as @endulab 's PR #6058

Yes, A part of the pr is already included in #6058. I am going to merge #6058 today, so this pr will have to be slightly updated.

**Review by jrainville — APPROVED** [review]

**Review by Hbouaz — APPROVED** [review]

**Pull request merged**